internal/worker/caasapplicationprovisioner/ops.go

```diff
@@ -210,7 +210,7 @@ func appAlive(appName string, app caas.Application, password string, lastApplied
 		Containers:           containers,
 		CharmModifiedVersion: provisionInfo.CharmModifiedVersion,
 		Trust:                provisionInfo.Trust,
-		InitialScale:         provisionInfo.Scale,
+		InitialScale:         0,
```

Comment:
With this change, `provisionInfo.Scale` is no longer used. We're still setting `provisionInfo.Scale` to `app.GetScale()` which is the app DesiredScale value. At deploy time, we set desired scale to 1 - remind me, we will see this as a scale change and create the first pod later? In theory we could delete the `provisionInfo.Scale` attribute it it's no longer used.